unstable/wlr-output-power-management-unstable-v1.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<protocol name="wlr_output_power_management_v1">
```

Comment:
This should be named `wlr_output_power_management_unstable_v1`

Reply:
fixed.